packages/xdg_directories/example/integration_test/xdg_directories_test.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' show Platform;
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:xdg_directories/xdg_directories.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('XDG Directories', (WidgetTester _) async {
+    // Check that the test is running on Linux.
+    if (!Platform.isLinux) {
+      throw Exception('This test is only valid on Linux');
+    }
+
+    // Check that getUserDirectory() returns a Directory.
+    expect(
+      getUserDirectory('Home') != null,
```

Comment:
This isn't testing the example, it's testing the library, which duplicates the unit tests (but much more expensively). The example test should be testing that the example itself (i.e., its widgets) are working correctly.

Reply:
I understand, thanks I'll remove these tests and add widget tests.

Reply:
I created widget tests for each text display.